core/server/worker/src/main/java/alluxio/worker/block/management/DefaultStoreLoadTracker.java

```diff
@@ -96,6 +101,8 @@ public void writerClosed(BlockWriter writer, BlockStoreLocation location) {
    * Used to activate stream reader/writer for load tracking.
    */
   private void streamOpened(Object stream, BlockStoreLocation location) {
+    LOG.debug("Stream opened. Type: {}, Id: {}, Location: {}",
+        stream.getClass().getSimpleName(), System.identityHashCode(stream), location);
```

Comment:
can we encapsulate this into some kind of object so that we can keep the open/close messages consistent?